com.github.kmkolasinski.AwesomeBump.json

```diff
@@ -0,0 +1,46 @@
+{
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "separate-locales": false,
+    "app-id": "com.github.kmkolasinski.AwesomeBump",
+    "command": "awesomebump",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "AwesomeBump",
+            "no-autogen": true,
+            "no-make-install": false,
+            "no-parallel-make": false,
+            "sources": [
+                {                    
+                    "tag": "Linuxv5.1.1",
+                    "commit": "d6912bfcb5219844841c2582f799c9a6dddaf533",
+                    "type": "git",
+                    "url": "https://github.com/kmkolasinski/AwesomeBump"
+                },
+                {
+                    "commands": [
+                       "qmake ./AwesomeBump.pro", "make -j 8", 
+                       "rm .qmake.stash", "make clean", "rm Makefile Sources/Makefile", 
+                       "qmake ./AwesomeBump.pro 'CONFIG+=gl330'", "make -j 8"
```

Comment:
Maybe instead of doing this tricky stuff, just build AwesomeBump twice, as separate modules, with plain qmake buildsystem? Something like this:
```json
{
    "name": "AwesomeBump",
    "buildsystem": "qmake",
    "sources": "..."
},
{
    "name": "AwesomeBump-gl3",
    "buildsystem": "qmake",
    "config-opts": [
        "CONFIG+=gl330"
    ],
    "sources": "..."
}
```

Reply:
Hi, firstly I want to make this manifest passing the build pipeline. Once this will work I can do some refactoring to make things more readable. I know that current manifest is not elegant, but at least more compact.